backend/src/main/java/com/zpi/backend/user_opinion/ReturnUserOpinionDTO.java

```diff
@@ -1,18 +1,21 @@
 package com.zpi.backend.user_opinion;
+import com.zpi.backend.user.User;
 import lombok.Data;
 import java.util.Date;
 
 @Data
-public class ReturnOpinionServiceDTO {
+public class ReturnUserOpinionDTO {
     private Date date;
     private String stars;
     private String description;
+    private User ratingUser;
```

Comment:
It cannot be User (it has google id, id, role atrributes). You can use for example GetUserDTO.

Reply:
Or better option would be create new DTO class with 
```
package com.zpi.backend.user;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class UserGameInstanceDTO {
    private String uuid;
    private String firstName;
    private String lastName;
    private String avatarLink;

    public UserGameInstanceDTO(User user){
        uuid = user.getUuid();
        firstName = user.getFirstName();
        lastName = user.getLastName();
        avatarLink = user.getAvatarLink();
    ```
    
   You can name it UserOpinionDTO and I'm going to rename it in Game Instance Opinion